src/utils.jl

```diff
@@ -21,25 +21,55 @@ equal probability is assigned to each entry in the support.
 
 # Examples
 ```julia
-using KernelFunctions
-# rows correspond to samples
-μ = discretemeasure(RowVecs(rand(7,3)), normalize!(rand(10),1))
 
-# columns correspond to samples, each with equal probability
-ν = discretemeasure(ColVecs(rand(3,12)))
+μ = discretemeasure(rand(10), normalize!(rand(10),1))
+
+# each entry has equal probability
+ν = discretemeasure(rand(3))
 ```
 
 !!! note
     If `support` is a 1D vector, the constructed measure will be sorted,
     e.g. for `mu = discretemeasure([3, 1, 2],[0.5, 0.2, 0.3])`, then
     `mu.support` will be `[1, 2, 3]` and `mu.p` will be `[0.2, 0.3, 0.5]`.
+    Also, avoid passing 1D distributions as something like `[[3],[1],[2]]`
+    since this will be dispatched to the multivariate case instead
+    of the univariate case for which the algorithm is more efficient.
```

Comment:
Is there actually a risk of someone passing `[[3], [1], [2]]`? I think this would be very very weird. Since I think we might still change this function and/or its functionality and return types, it would be good to add a warning such as
```suggestion

!!! warning
    This function and in particular its return values are not stable and might be changed in future releases.
```

Reply:
It might look silly, but I myself did it when using, cause I had `RowVecs(rand(10,2))`, so when I decided to do a 1D case, I just deleted the 2, thus `RowVecs(rand(10))`, which resulted in the "odd", yet possible case.